src/template_utils.rs

```diff
@@ -0,0 +1,126 @@
+use plume_models::{Connection, users::User};
+use rocket_i18n::Catalog;
+use templates::Html;
+
+pub type BaseContext<'a> = &'a(&'a Connection, &'a Catalog, Option<User>);
+
+pub enum Size {
+    Small,
+    Medium,
+}
+
+impl Size {
+    fn as_str(&self) -> &'static str {
+        match self {
+            Size::Small => "small",
+            Size::Medium => "medium",
+        }
+    }
+}
+
+pub fn avatar(conn: &Connection, user: &User, size: Size, pad: bool, catalog: &Catalog) -> Html<String> {
+    Html(format!(
```

Comment:
I haven't tested yet, but big warning on fields like `title`, one could maybe use a name containing html (or js) and get it unescaped in his page

Reply:
I was right on this one `"><script>alert("xss");</script> <div title="` as a display name trigger it very